api.py

```diff
@@ -34,9 +34,19 @@ class ApiVersion(str, Enum):
 config["description"] = os.getenv("DESCRIPTION", config.get("description", ""))
 config["debug"] = str(os.getenv("DEBUG", config.get("debug", False))).lower() in ("true", "1", "t")
 
+ELASTICSEARCH_INDEX_NAME_PREFIX = os.getenv("ELASTICSEARCH_INDEX_NAME_PREFIX")
+
 ES = Elasticsearch(config["eshosts"], **config["esopts"])
 
-Collection = list_to_enum("Collection", config["indexes"])
+
+def get_allowed_collections():
+    #Only expose indexes with the correct prefix, and add a wildcard as well. 
+    all_indexes = [index for index in ES.indices.get(index='*') if ELASTICSEARCH_INDEX_NAME_PREFIX in index]
+    all_indexes.extend([f"{ELASTICSEARCH_INDEX_NAME_PREFIX}_*)"])
```

Comment:
```suggestion
    all_indexes.append(f"{ELASTICSEARCH_INDEX_NAME_PREFIX}_*)")
```
I take we're are only adding this one item.